### DISCUSSION TITLE
Error displaying obtaining anittedata from V3 server while using sidestore

### DISCUSSION THREAD

**Discussion by moyu955** [SideStore/SideStore#643] [Q&A] [answered]
I am displaying the following paragraph when refreshing the application using sidestore
"An error occurred when getting anisettedata from a V3 server: Couldn't fetch clientinfo. The server may be down (too manyHTTP redirects). Please try again. lf the issuepersists, report it on GitHub lssues!"
![微信图片_20240620103842](https://github.com/SideStore/SideStore/assets/173326045/77b156c0-260c-4094-a077-aca169791111)

**Comment by SternXD** [comment, marked as answer]
Make a GitHub Issue this is for Discussions.
